.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+<!--
+Thank you for contributing to `too-many-hooks`! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to an issue) and what changes you've made, we will be best able to review your PR.
+-->
+
+### Why:
+
+Closes ISSUE
+
+<!-- If there's an existing issue for your change, please replace ISSUE above with a link to the issue.
+If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/yobgob/too-many-hooks/issues/new. -->
```

Comment:
Not sure that we need this rule. If someone wants to open a PR, I don't think a corresponding issue is necessary as long as the rationale is fully conveyed.